### ISSUE TITLE
Ollier type Enchondromatosis

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7110]
**Preferred term label:**
Ollier type Enchondromatosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

Text below from https://medlineplus.gov/genetics/condition/ollier-disease/  which has references.

Ollier disease is a disorder characterized by multiple enchondromas, which are noncancerous (benign) growths of cartilage that develop within the bones. These growths most commonly occur in the limb bones, especially in the bones of the hands and feet; however, they may also occur in the skull, ribs, and bones of the spine (vertebrae). Enchondromas may result in severe bone deformities, shortening of the limbs, and fractures.

The signs and symptoms of Ollier disease may be detectable at birth, although they generally do not become apparent until around the age of 5. Enchondromas develop near the ends of bones, where normal growth occurs, and they frequently stop forming after affected individuals stop growing in early adulthood. As a result of the bone deformities associated with Ollier disease, people with this disorder generally have short stature and underdeveloped muscles.

Although the enchondromas associated with Ollier disease start out as benign, they may become cancerous (malignant). In particular, affected individuals may develop bone cancers called chondrosarcomas, especially in the skull. People with Ollier disease also have an increased risk of other cancers, such as ovarian cancer or liver cancer.

**Parent term (use hpo.jax.org/app)**
HP:0030038

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ollier Syndrome

**Your nano-attribution (ORCID)**
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
@taylordm to my knowledge the enchondromas of Ollier disease are not different as phenotypic features from other enchondromas. This is reflected in the OMIM clinical synopsis, https://omim.org/entry/166000?search=ollier&highlight=ollier and the Orphanet annotation for Multiple enchondromatosis

https://hpo.jax.org/app/browse/disease/ORPHA:296

The fact that the enchondroma may undergo malignant malformation -- this would need to be coded with a phenopacket over time.

I am not an expert on Ollier, do you know if there are true phenotypic differences?

**Comment by pnrobinson** [comment]
@taylordm I think that this term is probably a Mondo term and not an HPO term. Please reopen if you disagree and we can work on a definition.
